### ISSUE TITLE
0.8.1 Not updating beta

### ISSUE THREAD

**Issue by motey** [shokinn/emby-updater#4]
Hey, first of, thanks for this great script :) 
i got an issue, where the updater is deciding that there is no newer version worth updating, but there is.

```sh
root@myserver:~# emby-updater --version
emby-updater 0.8.1
root@myserver:~# emby-updater --beta

NO UPDATE AVAILABLE!
Installed version:    4.5.0.9
Available version:    4.5.0.16-beta
```

i guess it has something to do that the current version is not flagged as `beta` -> `Installed version:    4.5.0.9`

But in Emby itself, the currently installed version is actually is flagged as beta.

![image](https://user-images.githubusercontent.com/1006417/89950073-2920b280-dc29-11ea-9c21-8ef741a66433.png)

Anybody else, or any idea what could cause this and how to fix it?

**Comment by shokinn** [comment]
Hey @motey,
I'm looking into your issue.
Unfortunately I'm not able to reproduce you bug due to removed binary distributions.
It looks like that the Emby devs remove the .deb files after they release 2 beta versions.
If you still have the 4.5.0.9 .deb file I'm happy to take again a look into it.

Sorry that I can't help here -.-

**Comment by shokinn** [comment]
Maybe @LukePulverenti can help here and send me the correct version?

**Comment by shokinn** [comment]
> i guess it has something to do that the current version is not flagged as beta -> Installed version: 4.5.0.9

Ahh and something to add here, this should not be a problem since the `tag_name` is used to compare versions and `tag_name` does not include the `-beta` suffix. So I just compare the version strings `4.5.0.9` and `4.5.0.16`.

**Comment by motey** [comment]
@shokinn Thanks a lot for having  a look. Do i understand you correctly; you think is a `4.5.0.9`specific bug? If i manually upgrade, i should be back on track?

Nevertheless, i will have a look into my server if i can find the deb file there after work.

**Comment by shokinn** [comment]
@motey,
jupp looks like.

I tried it today with `4.5.0.20` and `4.5.0.21` and the updater offered me to update emby.

See here:  
```
root@efa0d42de84d:/emby-updater# python3 -m embyupdater --beta
There is an update available
Installed version:    4.5.0.20
Available version:    4.5.0.21-beta
Do you want to update? (y/n):
```

**Comment by motey** [comment]
The old deb is lost, as is it saved in `/tmp` by default, correct? see https://github.com/shokinn/emby-updater/blob/3150289b17641df91bee45250843a5ea4452ed5c/embyupdater/__init__.py#L189

Thanks for the effort, but dont waste anymore time on that. Updated manually now and im fine :) 

For anyone with the same issue on a debian based system:
* Go to https://github.com/MediaBrowser/Emby.Releases/releases/tag/4.5.0.20 and find the fitting architecture. For most people amd64 will be the correct one. (go to https://github.com/MediaBrowser/Emby.Releases/releases/tag/4.4.3.0 for none beta releases)
* cd /tmp
* wget the binary via the link `wget https://github.com/MediaBrowser/Emby.Releases/releases/download/4.5.0.20/emby-server-deb_4.5.0.20_amd64.deb`
* install the binary `dpkg -i emby-server-deb_4.5.0.20_amd64.deb`
* `emby-updater` should be back on track
